gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPlugin.groovy

```diff
@@ -11,7 +11,16 @@ public class AndroidCommandPlugin implements Plugin<Project> {
         if (!project.plugins.hasPlugin('android')) {
             throw new StopExecutionException("The 'android' plugin is required.")
         }
-        def extension = project.android.extensions.create("command", AndroidCommandPluginExtension, project)
+
+        def android = project.android
+        def androidHome
+        if (android.hasProperty('sdkHandler')) {
+            androidHome = "${android.sdkHandler.sdkFolder}"
```

Comment:
Is this GString dance necessary? Might consider the to use the type of `sdkFolder` / `sdkDirectory`. Is is `File`?